frontend/common/src/components/SkillPicker/SkillBlock.tsx

```diff
@@ -91,21 +91,22 @@ const SkillBlock = ({
                 ? intl.formatMessage(
                     {
                       defaultMessage:
-                        "Remove skill<hidden> {skillName}</hidden>",
-                      id: "p5GbCr",
+                        "Remove this skill<hidden> {skillName}</hidden>",
+                      id: "hgipIO",
                       description:
-                        "Button label to remove skill on skill result block.",
+                        "Button label to remove the adjacent skill on skill result block.",
                     },
                     {
                       skillName,
                     },
                   )
                 : intl.formatMessage(
                     {
-                      defaultMessage: "Add skill<hidden> {skillName}</hidden>",
-                      id: "HB2yOT",
+                      defaultMessage:
+                        "Add this skill<hidden> {skillName}</hidden>",
```

Comment:
I just thought, maybe it makes more grammatical sense to add a colon in English?

```tsx
  "Add this skill<hidden>: {skillName}</hidden>",
```

Reply:
I thought about using punctuation like a colon or parenthesis.  But if this is for a screen reader does that actually make any difference?

Reply:
If we want to be meticulous, yes 😜 . But like, it _hardly_ matters because it doesn't add much.

REF: https://www.deque.com/blog/dont-screen-readers-read-whats-screen-part-1-punctuation-typographic-symbols/#:~:text=NVDA%20pauses%20briefly%2C%20but%20does%20NOT%20read%20the%20colon

Reply:
Fair enough.  It doesn't make it _worse_ for screen readers and maybe someday it will make it better for them.  :shrug:

Reply:
Cool, I like that!
![image](https://user-images.githubusercontent.com/8978655/198607604-16eec897-45f6-4630-8367-3b992cac61da.png)

Ref: [Add colon](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4477/commits/c426f884a3226f19dfaaca99f99dfa8708aaf685)